ethereum/ts-scripts/relayer/shell/upgradeWormholeRelayerMainnet.sh

```diff
@@ -0,0 +1 @@
+npx tsx ./ts-scripts/relayer/wormholeRelayer/submitWormholeRelayerImplementation.ts
```

Comment:
Running the check networks script is helpful to know what accounts will be used and their balances.